lib/services/HDInsight2/lib/hDInsightManagementClient.js

```diff
@@ -4094,7 +4094,17 @@ var ClusterOperations = ( /** @lends ClusterOperations */ function() {
                     var kindInstance = kindValue;
                     clusterDefinitionInstance.clusterType = kindInstance;
                   }
-                  
+
+                  var componentVerionsInDefinition = clusterDefinitionValue['componentVersion'];
+                  if (componentVerionsInDefinition !== null && componentVerionsInDefinition !== undefined) {
+                      var componentVerionsInstance = {};
+                      for (var item in componentVerionsInDefinition) {
+                        componentVerionsInstance[item] = componentVerionsInDefinition[item];
+                      }
+
+                      clusterDefinitionInstance.componentVerions = componentVerionsInstance;
+                  }
+
```

Comment:
@voidfunction  Are you from the hdInsight team ? If so then please contact Shefali Vohra regarding this.